raster/r.support/main.c

```diff
@@ -112,28 +120,48 @@ int main(int argc, char *argv[])
     map_opt->type = TYPE_STRING;
     map_opt->required = NO;
     map_opt->gisprompt = "old,cell,raster";
+    map_opt->guisection = _("Import / export");
```

Comment:
Changing "Import / export" to  "Import / Export" looks better in my eyes.

Reply:
My Latvian language thinking always finds a way into my otherwise flawless and excellent Baltic English language :D